### PR TITLE
add RSS feed to resources

### LINKED ISSUE #2490: [Resources] RSS Feed: Create feed for all resources
**Current Behavior**
Site visitors can browse [resources](https://layer5.io/resources), but not subscribe for notifications of newly published resources.

**Desired Behavior**
Create `/resources/feed.xml` - an RSS feed containing all resources.

---
**Screenshots / Mockups**
Screen Shot 2022-01-16 at 8 48 44 PM

**Implementation**
1. Create new RSS feed in `gatsby-config.js` to be available at `/resources/feed.xml`.
1. Add RSS icon with link to feed adjacent to /resources page title as pictured in mockup.

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Sunnyio**
Signed-off-by: Sunny Kumar <sunny514173@gmail.com>

**Description**
Add RSS feed to all resources
This PR fixes #2490

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 47a71f4365c3fb5b83e19866cf79ad0118890435 at: https://61eacdf304a67f0b71a5df49--layer5ng.netlify.app

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍

**Review by YashKamboj — APPROVED** [review]
LGTM!!!!

**Comment by subhradwip01** [comment]
Actually, It is opening the layer5 blog RSS feed when we click on the RSS feed icon...change the feed link...then it will work fine

**Comment by Abhijay007** [comment]
> Actually, It is opening the layer5 blog RSS feed when we click on the RSS feed icon...change the feed link...then it will work fine😇

nice observation @subhradwip01

File: src/pages/resources/index.js [hunk 1]
```diff
@@ -42,7 +43,7 @@ const MembersPage = () => {
         <GlobalStyle />
         <SEO title="Service Mesh Resources" description="Articles on how to service mesh from the world's largest service mesh community. Service mesh how-tos and cloud native ecosystem news." canonical="https://layer5.io/resources" />
         <Navigation />
-        <PageHeader title="Cloud Native Resources" path="Resources" subtitle="Learn how to provision, secure, connect, and run any Kubernetes and any service mesh for any application." />
+        <PageHeader title="Cloud Native Resources" path="Resources" subtitle="Learn how to provision, secure, connect, and run any Kubernetes and any service mesh for any application." img={RssFeedIcon} feedlink="/blog/feed.xml"/>
```
Comment (Abhijay007) [inline src/pages/resources/index.js:46]: @Sunnyio change feedlink here to : "/resources/feed.xml"

**Review by Abhijay007 — CHANGES_REQUESTED** [review]

File: src/pages/resources/index.js (hunk 1 above)
Comment (Sunnyio) [reply src/pages/resources/index.js:46]: my bad, changing it now.

Comment (Sunnyio) [reply src/pages/resources/index.js]: @Abhijay007  can you review it now?

**Comment by Sunnyio** [comment]
> Actually, It is opening the layer5 blog RSS feed when we click on the RSS feed icon...change the feed link...then it will work fine😇

hey @subhradwip01 can you review it again please.

**Comment by l5io** [comment]
🚀 Preview for commit 30e0cbe9af0ff788ffd26327db4d8e0f2da4cef0 at: https://61ec39f00b7692531f055066--layer5ng.netlify.app

**Comment by subhradwip01** [comment]
> > Actually, It is opening the layer5 blog RSS feed when we click on the RSS feed icon...change the feed link...then it will work fine😇
> 
> hey @subhradwip01 can you review it again please.

Yes now it's looking good 😊

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍

File: src/pages/resources/index.js [hunk 2]
```diff
@@ -42,7 +43,7 @@ const MembersPage = () => {
         <GlobalStyle />
         <SEO title="Service Mesh Resources" description="Articles on how to service mesh from the world's largest service mesh community. Service mesh how-tos and cloud native ecosystem news." canonical="https://layer5.io/resources" />
         <Navigation />
-        <PageHeader title="Cloud Native Resources" path="Resources" subtitle="Learn how to provision, secure, connect, and run any Kubernetes and any service mesh for any application." />
+        <PageHeader title="Cloud Native Resources" path="Resources" subtitle="Learn how to provision, secure, connect, and run any Kubernetes and any service mesh for any application." img={RssFeedIcon} feedlink="/resources/feed.xml"/>
```
Comment (leecalcote) [inline src/pages/resources/index.js:46]:
```suggestion
        <PageHeader title="Cloud Native Resources" path="Resources" subtitle="Learn how to provision, secure, connect, and manage your applications on Kubernetes and any service mesh" img={RssFeedIcon} feedlink="/resources/feed.xml"/>
```

**Comment by l5io** [comment]
🚀 Preview for commit 84cdcb30da529f6636446a46d7f69ebb32c9e9dd at: https://61ec9baf6a86fce98002653a--layer5ng.netlify.app

File: gatsby-config.js [hunk 3]
```diff
@@ -145,6 +145,54 @@ module.exports = {
...
+                    fields {
+                      collection
+                      slug
+                    }
+                  }
+                }
+              }
+            `,
+            output: "/resources/feed.xml",
+            title: "Layer5",
```
Comment (leecalcote) [inline gatsby-config.js:194]:
```suggestion
            title: "Layer5 Blog",
```

**Review by leecalcote — APPROVED** [review]
Looking good. 👍

**Comment by leecalcote** [comment]
yay, @Sunnyio 🎈

**Pull request merged**